packages/react/src/components/TimePicker/_time-picker-dropdown.scss

```diff
@@ -66,14 +66,17 @@
 
     .#{$iot-prefix}--time-picker__wrapper-sm & {
       bottom: 1rem;
+      padding-right: 0;
```

Comment:
I believe that this element already has no padding applied. So these padding declarations are unnecessary.

Reply:
Moved the padding-right out of <TimePickerDropdown>.